sites.php

```diff
@@ -328,6 +330,18 @@ function jquery_sites() {
 				'active_plugins' => array(),
 			),
 		),
+		'releases.jquery.com' => array(
+			'blog_id' => 29,
+			'cookie_domain' => '.jquery.com',
+			'body_class' => 'jquery',
+			'options' => array(
+				'blogname' => 'jQuery Releases',
```

Comment:
nitpick: Could we keep the name "jQuery CDN" for the portal?

Reply:
We can, but it will not be 100% true as `*-git.*` files will not be served from the CDN.

Reply:
...but I see that `jQuery Releases` may be seen as referring to Core releases only.

Reply:
Updated.